test/tests/functional/pbs_equiv_classes.py

```diff
@@ -0,0 +1,1052 @@
+        Return a list of job ids
+        """
+        ret_jids = []
+        for n in range(num_jobs):
+            J = Job(user, attrs)
+            jid = self.server.submit(J)
+            ret_jids += [jid]
+
+        return ret_jids
+
+    def test_basic(self):
+        """
+        Test the basic behavior of job equivalence classes: submit two
+        different types of jobs and see they are in two different classes
+        """
+        self.server.manager(MGR_CMD_SET, MGR_OBJ_SERVER,
+                            {'scheduling': 'False'})
+
+        # Eat up all the resources
+        a = {'Resource_List.select': '1:ncpus=8'}
+        J = Job(TEST_USER, attrs=a)
+        self.server.submit(J)
+
+        jids1 = self.submit_jobs(3, a)
+
+        a = {'Resource_List.select': '1:ncpus=4'}
+        jids2 = self.submit_jobs(3, a)
+
+        self.server.manager(MGR_CMD_SET, MGR_OBJ_SERVER,
+                            {'scheduling': 'True'})
+
+        m = self.scheduler.log_match("Number of job equivalence classes: 2",
+                                     max_attempts=10, starttime=self.t)
+        self.assertTrue(m)
+
+    def test_select(self):
+        """
+        Test to see if jobs with select resources not in the resources line
+        fall into the same equivalence class
+        """
+        self.server.manager(MGR_CMD_CREATE, MGR_OBJ_RSC,
+                            {'type': 'long', 'flag': 'nh'}, id='foo')
+
+        # Eat up all the resources
+        a = {'Resource_List.select': '1:ncpus=8'}
+        J = Job(TEST_USER, attrs=a)
+        self.server.submit(J)
+
+        a = {'Resource_List.select': '1:ncpus=1:foo=4'}
+        jids1 = self.submit_jobs(3, a)
+
+        a = {'Resource_List.select': '1:ncpus=1:foo=8'}
+        jids2 = self.submit_jobs(3, a)
+
+        self.server.manager(MGR_CMD_SET, MGR_OBJ_SERVER,
+                            {'scheduling': 'True'})
+
+        # Two equivalence classes: one for the resource eating job and one
+        # for the other two jobs. While jobs have different amounts of
+        # the foo resource, foo is not on the resources line.
+        m = self.scheduler.log_match("Number of job equivalence classes: 2",
+                                     max_attempts=10, starttime=self.t)
+        self.assertTrue(m)
+
+    def test_place(self):
+        """
+        Test to see if jobs with different place statements
+        fall into the different equivalence classes
+        """
+        # Eat up all the resources
+        a = {'Resource_List.select': '1:ncpus=8'}
+        J = Job(TEST_USER, attrs=a)
+        self.server.submit(J)
+
+        a = {'Resource_List.select': '1:ncpus=1',
+             'Resource_List.place': 'free'}
+        jids1 = self.submit_jobs(3, a)
+
+        a = {'Resource_List.select': '1:ncpus=1',
+             'Resource_List.place': 'excl'}
+        jids2 = self.submit_jobs(3, a)
+
+        self.server.manager(MGR_CMD_SET, MGR_OBJ_SERVER,
```

Comment:
I hadn't really noticed the difference.  Both work.  I did a grep through the tests and didn't find any other that used the MGR_OBJ versions, so you're right.  I removed the MGR_OBJ from all of the manager() calls.